### PR TITLE
Update README.md file 4

### PR DESCRIPTION
**Pull request by rulalub**
new test repo file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+In this file, I will learn markdown. I will learn it by:
+
+* Practicing
+* Trial and error
+* Copying examples and seeing their output.
+
+I will use [this reference](https://guides.github.com/features/mastering-markdown/) as a guide.
+
+One day **soon** I will be a
```
Comment (lsnovich) [inline README.md]:
```suggestion
One day **soon** I will be a markdown master!
```

Comment (rulalub) [reply README.md]: remove the last part

**Comment by lsnovich** [comment]
Two things
1 - There is a suggestion I gave you as you didn't' paste the entire text block. 
2 - You have a conflict that needs to be resolved before you merge.

**Comment by lsnovich** [comment]
all good to go!
You are clear to merge!

**Pull request merged**